Tests/TensorFlowTests/OperatorTests/MathTests.swift

```diff
@@ -199,6 +199,13 @@ final class MathOperatorTests: XCTestCase {
         let output: Float = 1.0
         XCTAssertEqual(z, Tensor(output))
     }
+    func testCosineDistance(){
```

Comment:
Can you add an empty line before the function? Also 
```suggestion
    func testCosineDistance() {
```